rts/lra/lra-client/src/main/java/io/narayana/lra/client/NarayanaLRAClient.java

```diff
@@ -548,18 +555,33 @@ public String joinLRA(URL lraId, Class<?> resourceClass, URI baseUri,
         return null;
     }
 
-    @Override
-    public URL updateCompensator(URL recoveryUrl, URL compensateUrl, URL completeUrl,
-                                 URL forgetUrl, URL statusUrl,
-                                 String compensatorData) throws GenericLRAException {
-        return null;// TODO
+    public URL updateCompensator(URL url, Class<?> aClass, URI uri, String s) throws GenericLRAException {
+        return null; // TODO
```

Comment:
I expect there should not be left this `// TODO` and some issue should be created instead.

Reply:
The JIRA was to make it work with the latest microprofile-lra spec updates so I will raise a separate JIRA.

Reply:
I hesistate a bit. My understanding of the correct handling this in manner of how the [clean code](https://www.amazon.com/Clean-Code-Handbook-Software-Craftsmanship/dp/0132350882) defends it is - remove the `// TODO` now and create a new jira that would directly talk on what needs to be fixed here. There will be reasoning about why this todo is here now particularly. The `// TODO` says nothing now. The jira should clarifies the issue.

Reply:
Disagree. This PR is to update the code to make it work with the latest MP TCK - it is not to go through the LRA code base cleaning up TODOs and comments etc. We have a policy of not making updates to code in a PR unless it specifically relates to the JIRA for which the PR was raised against. What you are asking for should be fixed in a separate PR for JBTM-3089.

Reply:
From what I can see the TODO already existed right?

Reply:
Correct, that is the point I am trying to get Ondra to understand.